Dockerfile

```diff
@@ -1,61 +1,59 @@
-FROM phusion/baseimage:0.9.15
+FROM phusion/baseimage:0.9.18
 MAINTAINER Open Knowledge
 
-# Disable SSH
-RUN rm -rf /etc/service/sshd /etc/my_init.d/00_regen_ssh_host_keys.sh
-
-ENV HOME /root
+# Environment
 ENV CKAN_HOME /usr/lib/ckan/default
 ENV CKAN_CONFIG /etc/ckan/default
 ENV CKAN_DATA /var/lib/ckan
+ENV CKAN_GIT_BRANCH release-v2.5.2
```

Comment:
The idea, is give the feature of change the deployed branch. This makes the docker image ready for production. If somebody wants to change to another release(branch of ckan/ckan, master included), they only have to change that ENV.

But there are a know issue:
If you build or pull the docker file and ENV is release-v2.5.2, and then you change that ENV to for example "master", there will be no changes, because docker cache every layer[https://github.com/docker/docker/issues/3199(https://github.com/docker/docker/issues/3199)

The solution, is to use "build --no-cache"

I think I should add documentation about how to use it, this is safe for production